packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -311,19 +324,39 @@ LiveLabel.defaultProps = {
   dir: 'ltr',
 };
 
-const StoryPromo = ({ image, info, mediaIndicator, topStory, ...props }) => (
+const StoryPromo = ({
+  image,
+  info,
+  mediaIndicator,
+  topStory,
+  displayImage,
+  ...props
+}) => (
   <StoryPromoWrapper topStory={topStory} {...props}>
-    <ImageGridItem topStory={topStory}>
-      <ImageContentsWrapper>
-        {image}
-        {mediaIndicator && (
-          <InlineMediaIndicator topStory={topStory}>
-            {mediaIndicator}
-          </InlineMediaIndicator>
-        )}
-      </ImageContentsWrapper>
-    </ImageGridItem>
-    <TextGridItem topStory={topStory}>{info}</TextGridItem>
+    {displayImage ? (
+      <>
+        <ImageGridItem topStory={topStory}>
+          <ImageContentsWrapper>
+            {image}
+            {mediaIndicator && (
+              <InlineMediaIndicator topStory={topStory}>
+                {mediaIndicator}
+              </InlineMediaIndicator>
+            )}
+          </ImageContentsWrapper>
+        </ImageGridItem>
+        <TextGridItem topStory={topStory} displayImage={displayImage}>
+          {info}
+        </TextGridItem>
+      </>
+    ) : (
+      <>
+        <TextGridItem topStory={topStory} displayImage={displayImage}>
+          {mediaIndicator}
+          {info}
+        </TextGridItem>
+      </>
+    )}
```

Comment:
Would this be easier to read if it were something more like this?
```jsx
const image = displayImage && (
       <ImageGridItem topStory={topStory}
          <ImageContentsWrapper>
            {image}
            {mediaIndicator && (
              <InlineMediaIndicator topStory={topStory}>
                {mediaIndicator}
              </InlineMediaIndicator>
            )}
          </ImageContentsWrapper>
        </ImageGridItem>
      )

    <>
      {image}
      <TextGridItem topStory={topStory} displayImage={displayImage}>
        {info}
      </TextGridItem>
    </>
```
Maybe with `image` being a separate component so it could be `const image = displayImage && NewImageComponent`

Reply:
Good idea, that is easier to read and looks cleaner